cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/CardDataDTO.java

```diff
@@ -10,7 +10,11 @@
 import java.util.HashMap;
 import java.util.Map;
 
+/**
+ * @deprecated by {@link TokenDataDTO}
+ */
 @ApiModel( description = "Credit card data" )
+@Deprecated
 public class CardDataDTO extends PaymentDataDTO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)